### PR TITLE
NAS 100046

### PR DESCRIPTION
**Pull request by lola27**

### CODE REVIEW

**Comment by lola27** [comment]
@wblock @freenas/docs-team the tooltips for initiators and authorized network needs an update. we don't use keyword 'ALL' anymore (use the checkbox instead), and don't allow to input more than one network address at once, need to add one by one.

**Comment by Mrt134** [comment]
@lola27 Warren is out today, so I went ahead and updated the helptext for these initiators fields based on your comments and one of his offline comments.

File: src/app/helptext/sharing/iscsi/iscsi.ts [hunk 1]
```diff
@@ -81,23 +81,30 @@ export const helptext_sharing_iscsi = {
...
 
   initiator_form_placeholder_auth_network: T("Authorized Networks"),
   initiator_form_tooltip_auth_network: T(
-    'Network addresses which can use this initiator. Use\
- <i>ALL</i> or list network addresses with a\
- <a href="https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing"\
- target="_blank">CIDR</a> mask. Separate multiple\
- addresses with a space:\
- <i>192.168.2.0/24 192.168.2.1/12</i>.'
+    'Network addresses which can use this initiator. Enter an IPv4 or\
```
Comment (wblock) [inline src/app/helptext/sharing/iscsi/iscsi.ts]: What happened to the updated text about the optional CIDR netmask?

Comment (Mrt134) [reply src/app/helptext/sharing/iscsi/iscsi.ts]: Got ripped out when I rewrote the messages. I'll add it back.

**Pull request merged**